exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,11 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+const students = ;
```

Comment:
You assigned variable students to nothing, when I run this exercise I get a syntax error because of this line.

Reply:
I forgot to solve it